README.md

```diff
@@ -183,14 +183,17 @@ They live in [tests](tests). Run them with grunt: `grunt e2e`.
 
 ## Configuring Medic
 
-We ship with one "standard" configuration, which can be a useful basis to start with. It is located at [./config/standard](https://github.com/medic/medic/tree/master/config/standard).
+We ship with one "standard" configuration, which can be a useful basis to start with. It is located at [./config/standard](https://github.com/medic/medic/tree/master/config/standard). 
+
+This app is highly configurable and can be modified to suit your needs. Read the guide for developing community health applications if you would like to customize your application further.
```

Comment:
I'd put the second paragraph here above the first. Start with the role of configuration and then talk about the single specific configuration.

We don't really "ship with" a configuration. Really it is just "in this repository". I remember finding it really confusing until I realised that the other configurations were in other repositories. So maybe useful to mention?

Maybe link to the "guide for developing community health applications" ?